modules/covid_vaccine/spec/jobs/covid_vaccine/registration_email_job_spec.rb

```diff
@@ -24,12 +23,11 @@
     end
 
     it 'the service is initialized with the correct parameters with disabled toggle' do
```

Comment:
Is this test still needed if toggle is removed?

Reply:
merged the two test into one to check the correct api_key is used